### ISSUE TITLE
Userstory 2 

### ISSUE THREAD

**Issue by kisa5** [kisa5/capstone-project#4]
## Value Proposition

**As a** User
**I want to** click on my project
**so that** I can see all my on and forthcoming tasks

## Description

<img width="808" alt="Bildschirmfoto 2023-01-23 um 11 54 49" src="https://user-images.githubusercontent.com/116880659/214032431-478c6a1f-831e-4832-9f9a-f89b0f90db23.png">

## Acceptance Criteria

- [ ] if you click on a project card you will navigate to the project page 
- [ ] in the top left corner is a "back" button for navigating back to the home page
- [ ] on the project page the title is positioned fixed on top of the screen
- [ ] several tasks for the respective project are listet under the project title 

## Tasks

- [ ] for the project page create a new file in the pages folder
- [ ] in the project page file create a button with a onClick function for navigating back to the home page 
- [ ] use useRouter to implement dynamic routing for project pages
- [ ] fix the project title on top of the screen
- [ ] list the project tasks below the title
- [ ] on the homepage wrap the project cards in a link that navigates to the project page 

## Complexity

M

**Comment by RobinWitt** [comment]
Hi Anna,
I wonder what the checkboxes are doing at the point of this user story. If you are not adding any function to them yet, you should not mention them in the wireframe and user story tasks.

I think the checkbox implementation and functionality deserve their own user story.

If you cut them out, you have a well-rounded complexity and scope. I also like the descriptiveness of this story.
One more thing: Either don't mention that you import useRouter or be more specific on why you need it. (suggestion: "use useRouter to implement dynamic routing for project pages")

**Comment by kisa5** [comment]
Thanks for your review. Did an update :)

**Comment by RobinWitt** [comment]
Now it's perfect in my opinion!

**Comment by Long-HaHoang** [comment]
Hi Anna,
You have a typo in one of your fifth task. `unter` instead of `under`
But even so the wording is not right.
I would rephrase it.

- [ ] List the project tasks below the title

Just a minor change. Otherwise looks good to me with the changes Robin suggested. 👍

**Comment by kisa5** [comment]
Corrected it 😁 @Long-HaHoang
